src/keyValueStore.js

```diff
@@ -81,7 +82,11 @@ class KeyValueStore {
   }
 
   async _load (odbAddress) {
-    this._db = await this._orbitdb.keyvalue(odbAddress || this._name)
+    const key = this._3id.getKeyringBySpaceName(this._name).getDBKey()
+    this._db = await this._orbitdb.keyvalue(odbAddress || this._name, {
+      key,
+      write: [key.getPublic('hex')]
+    })
```

Comment:
is this adding the key as a signing key to store and giving it write permissions?

Reply:
vs init orbitdb took a keystore adapter before and did it for each store we initialized

Reply:
So the orbitdb keystore could only retreive keys by ipfs node id. So we could only have one key. This key was added with write permissions by default to all stores that where created without a `write` array. Since with spaces we need to have different keys for each space we need to add the signing key and set the write permissions.

Reply:
So yes, you are right 🙂

Reply:
ah yup, great, thanks for the extra details!